bastion/task/cmd/backfill.go

```diff
@@ -0,0 +1,91 @@
+/*
+Copyright © 2023 NAME HERE <EMAIL ADDRESS>
```

Comment:
?

Reply:
This was auto generated by cobra. Deleted this section for all the cobra generated files.